x86/boards.txt

```diff
@@ -1,7 +1,87 @@
-virtual.name="Kaleidoscope Virtual Keyboard"
```

Comment:
instead of having a virtual board. We will have a virtual core that is used by individual boards that represent the individual keyboards.